modules/understanding-agent-install.adoc

```diff
@@ -60,6 +60,11 @@ Recommended cluster resources for the following topologies:
 
 The following platforms are supported:
 
-* `none`
 * `baremetal`
 * `vsphere`
+* `none`
++
+[NOTE]
+====
+The `none` option is only supported for single-node OpenShift clusters with `NetworkType=OVNKubernetes`.
```

Comment:
s/only supported for/supported for only (the _only_ goes next to the term it is modifying)

s/with `NetworkType=OVNKubernetes`/of the `OVNKubernetes` network type. (currently, it seems a bit lacking in something)